app/src/ereferrals/java/org/eyeseetea/malariacare/data/remote/OrgUnitDataSource.java

```diff
@@ -0,0 +1,16 @@
+package org.eyeseetea.malariacare.data.remote;
+
+import org.eyeseetea.malariacare.domain.boundary.repositories.IOrgUnitRepository;
+import org.eyeseetea.malariacare.domain.entity.Credentials;
+import org.eyeseetea.malariacare.domain.entity.OrgUnit;
+import org.eyeseetea.malariacare.domain.exception.NetworkException;
+import org.eyeseetea.malariacare.domain.exception.PullConversionException;
+import org.eyeseetea.malariacare.network.ServerAPIController;
+
+public class OrgUnitDataSource implements IOrgUnitRepository {
```

Comment:
@manuelplazaspalacio After @ifoche merge this PR https://github.com/EyeSeeTea/pictureapp/pull/1074 There is already a OrganisationUnitDataSource with similar logic, can you refactor for unify code?